test-network-function/generic/suite.go

```diff
@@ -208,6 +209,10 @@ var _ = ginkgo.Describe(testsKey, func() {
 		for _, containersUnderTest := range containersUnderTest {
 			testDeployments(containersUnderTest.oc.GetPodNamespace())
 		}
+
+		for _, containersUnderTest := range containersUnderTest {
```

Comment:
I know there's several of these loops which are reusing `containersUnderTest` to refer to a single container inside the loop but please don't add another one. We'll clean up the others separately at a later time.

Reply:
so merge into an existing loop?

Reply:
I think I did not understand your first comment. What is the requested change?

Reply:
The term I was failing before to find was shadowing.
This `for` loop (and other similar ones in this file) is shadowing `containersUnderTest` inside the loop. This makes the code harder to understand and work with.
I'm not asking you to change all the occurrences of this (that should be done but will obfuscate the specifics of this change), rather not to add another one.